inventory/hosts

```diff
@@ -93,7 +93,7 @@ ie_prod
 # India
 
 [in_prod]
-openfoodnetwork.in
+openfoodnetwork.in ansible_user=ubuntu
```

Comment:
I think that this is only required to start with, or it was required before your key was added. Once your key is in the ofn-admin account, this user isn't needed any more. This setting will prevent other developers to access the server unless they override it with ofn-admin, I think. I'm not 100% sure about precedence.

Reply:
Yes, it is not required for `playbooks/deploy.yml` so I removed and I have successfully performed
```
ansible-playbook playbooks/deploy.yml --limit in_prod -e "git_version=v4.4.17"
```

Do you need to run setup also ?
When I try setup there is an error
```
ansible-playbook playbooks/setup.yml --limit in_prod -e "git_version=v4.4.17"

PLAY [python check] ****************************************************************************************************************

TASK [Install python] **************************************************************************************************************
Friday 03 November 2023  09:48:35 +0100 (0:00:00.017)       0:00:00.017 ******* 
fatal: [openfoodnetwork.in]: UNREACHABLE! => {"changed": false, "msg": "Failed to connect to the host via ssh: root@openfoodnetwork.in: Permission denied (publickey).", "unreachable": true}

PLAY RECAP *************************************************************************************************************************
openfoodnetwork.in         : ok=0    changed=0    unreachable=1    failed=0    skipped=0    rescued=0    ignored=0   
```
but probably you need only to deploy... Not sure if we need to add some system dependency or upgrade some package, probably I need to do that, right ?

Reply:
Hmm, it looks like the first-time setup playbook needs to log in as `root`, in order to create other users.

But I guess you've already executed the `setup.yml` playbook? In that case you don't need to run it again. 
All other playbooks (for example `provision.yml`) should still work, using the default user `ofn-admin`.

Reply:
@dacook  OK, we can keep without `ansible_user=ubuntu` since setup is already performed. Do you connect to other instances using `root` or only using those generated users `ofn-admin`, `openfoodnetwork` ?

Reply:
I think we only connect using those generated users. I'm not sure, but I think we usually disable SSH access for the root user.